presto-docs/src/main/sphinx/release/release-0.254.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.254
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where queries that have both remote functions and a local function with only constant arguments could throw an IndexOutOfBoundException during planning. The bug was introduced in release 0.253 by :pr:`16039`.
+* Add documentation for Glue Catalog support in Hive. :doc:`/connector/hive`.
+* Add iceberg connector.
+* Add support to fragment result caching for unnest.
```

Comment:
what does this mean?

Reply:
@shixuan-fan could you provide more details

Reply:
How about "Add fragment result caching support for UNNEST queries"?